source/config/__init__.py

```diff
@@ -914,6 +762,21 @@ def saveProfileTriggers(self):
 		self.triggersToProfiles.parent.write()
 		log.info("Profile triggers saved")
 
+	def getConfigValidationParameter(self, keyPath, validationParameter):
+		"""Get a config validation parameter
+		This can be used to get the min, max, default, or other values for a config key.
+		@param keyPath: a sequence of the identifiers leading to the config key. EG ("braille", "messageTimeout")
+		@param validationParameter: the parameter to return the value for. EG "max"
+		@type validationParameter: string
+		"""
+		if not keyPath or len(keyPath) < 1:
+			raise ValueError("Key path not provided")
+
+		spec = conf.spec
+		for nextKey in keyPath:
+			spec = spec[nextKey]
+		return conf.validator._parse_with_caching(spec)[2][validationParameter]

```

Comment:
I don't think this could be the cause of any current problems:

1. This is a new function that is only being used in one place right now (cursor blink rate) and that place doesn't involve a `__many__`.
2. Even though `__many__` is used for speech synths, synths build their own specific config spec which overrides this.